src/main/drivers/barometer/barometer.h

```diff
@@ -29,6 +29,7 @@ typedef void (*baroCalculateFuncPtr)(int32_t *pressure, int32_t *temperature); /
 
 typedef struct baroDev_s {
     busDevice_t busdev;
+    bool combined_read;
```

Comment:
There is a lot of seemingly unrelated change in here. Can you please clean this up so it can be reviewed?